### PR TITLE
build v1.1.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- noarch removed.
 - build script updated.
 - host dependencies updated.
 - test section updated.
 - description addede.
 - doc_url ammended.

### CODE REVIEW

**Comment by akabanovs** [comment]
abs.yaml will be added after review

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
this is the snowflake, so do not forget publish it to the snowlfake channel only, otherwise LGTM

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,16 +27,25 @@ requirements:
     - six
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:27]: `six` has been removed, see https://github.com/plotly/plotly.py/commit/c3fb1a575f009057111f0a7d149861fda2c59129#diff-0b2c14531dd70b010865c2b62e0d87e2896d90de0a2ed420fc0e9ff3b0218787

Comment (akabanovs) [reply recipe/meta.yaml:27]: @skupr-anaconda  hmm I used the setup.py file from the source code I downloaded from the pypi url in the recipe:
install_requires=["plotly", "requests", "retrying>=1.3.3", "six"]

```
 chart-studio-1.1.0 grep -r 'six' ./*
./chart_studio/tools.py:import six
./chart_studio/tools.py:    if isinstance(username, six.string_types):
./chart_studio/tools.py:    if isinstance(api_key, six.string_types):
./chart_studio/tools.py:    if isinstance(proxy_username, six.string_types):
./chart_studio/tools.py:    if isinstance(proxy_password, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_streaming_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_api_domain, six.string_types):
./chart_studio/tools.py:    if isinstance(plotly_ssl_verification, (six.string_types, bool)):
./chart_studio/tools.py:    if isinstance(plotly_proxy_authorization, (six.string_types, bool)):
./chart_studio/tools.py:    if isinstance(sharing, six.string_types):
./chart_studio/tools.py:        urlsplit = six.moves.urllib.parse.urlparse(url)
./chart_studio/tools.py:        query_dict = six.moves.urllib.parse.parse_qs(urlsplit.query)
./chart_studio/session.py:import six
./chart_studio/session.py:    "username": six.string_types,
./chart_studio/session.py:    "api_key": six.string_types,
./chart_studio/session.py:    "proxy_username": six.string_types,
./chart_studio/session.py:    "proxy_password": six.string_types,
./chart_studio/session.py:    "plotly_domain": six.string_types,
./chart_studio/session.py:    "plotly_streaming_domain": six.string_types,
./chart_studio/session.py:    "plotly_api_domain": six.string_types,
./chart_studio/session.py:    "sharing": six.string_types,
./chart_studio/session.py:    "filename": six.string_types,
./chart_studio/session.py:    "fileopt": six.string_types,
./chart_studio/session.py:    "sharing": six.string_types,
./chart_studio/plotly/chunked_requests/chunked_request.py:import six
./chart_studio/plotly/chunked_requests/chunked_request.py:from six.moves import http_client
./chart_studio/plotly/chunked_requests/chunked_request.py:from six.moves.urllib.parse import urlparse, unquote
./chart_studio/plotly/chunked_requests/chunked_request.py:        self._bytes = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:                return six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:            if _bytes == six.b(""):
./chart_studio/plotly/chunked_requests/chunked_request.py:        if response != six.b(""):
./chart_studio/plotly/chunked_requests/chunked_request.py:                response = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:            self._bytes = six.b("")
./chart_studio/plotly/chunked_requests/chunked_request.py:class _FakeSocket(six.StringIO):
```

Comment (akabanovs) [reply recipe/meta.yaml:27]: So pypi chart-studio seems to have the source files that still require six package

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**